ci/tests/20-fputil.bash

```diff
@@ -11,11 +11,17 @@ export SCRIPT_DIR="$(dirname ${BASH_SOURCE})/.."
 # Loop over deployments and targets
 for deployment in ${FPUTIL_DEPLOYS}
 do
+    export CMAKE_EXTRA_SETTINGS=""
     if [[ "${deployment}" == */RPI ]] && [ ! -d "/opt/rpi/tools/arm-bcm2708/arm-rpi-4.9.3-linux-gnueabihf" ]
     then
         warn_and_cont "RPI tools not installed, refusing to test."
         continue
     fi
+    # Check not RPI and Ref deployment to enable FRAMEWORK UTS
+    if [[ "${deployment}" != */RPI ]] && [[ "${deployment}" != */Ref ]]
+    then
+        export CMAKE_EXTRA_SETTINGS="${CMAKE_EXTRA_SETTINGS} -DFPRIME_ENABLE_FRAMEWORK_UTS=ON"
```

Comment:
We should enable autocoder unit tests for the default build as well.

Reply:
Autocoder UTs are by default on with DFPRIME_ENABLE_FRAMEWORK_UTS=ON and must be shutoff with -DFPRIME_ENABLE_FRAMEWORK_UTS=OFF